docs/csharp/language-reference/compiler-options/how-to-set-environment-variables-for-the-visual-studio-command-line.md

```diff
@@ -35,7 +35,11 @@ If the current version of Visual Studio is installed on a computer that also has
   
     > [!CAUTION]
     >  VsDevCmd.bat can vary from computer to computer. Do not replace a missing or damaged VsDevCmd.bat file with a VsDevCmd.bat from another computer. Instead, rerun setup to replace the missing file.  
-  
+
+### Available options for VsDevCmd.BAT
+
+To see the available options for VsDevCmd.BAT run the command with the `-help` flag, i.e. `VsDevCmd.bat -help`.
```

Comment:
Two nits:
- comma before "run"
- Rather than "flag, i.e., \`VsDevCmd.bat -help\`.

   > flag:
   > \`\`\`console
   > VsDevCmd.bat -help
   > \`\`\`
  
    That allows the command to be copied-and-pasted easily.

Reply:
Corrected